cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -174,8 +194,16 @@ def index_item(item, skip_index=False):
         try:
             with modulestore.branch_setting(ModuleStoreEnum.RevisionOption.published_only):
                 structure = cls._fetch_top_level(modulestore, structure_key)
+                groups_usage_info = None
+                if cls.DOCUMENT_TYPE == "courseware_content":
+                    groups_usage_info = GroupConfiguration.get_content_groups_usage_info(modulestore, structure).items()
+                    if groups_usage_info:
+                        for name, group in groups_usage_info:  # pylint: disable=unused-variable
```

Comment:
Should this be rewritten in the following fashion `for group in groups_usage_info.values():` because `name` is unused here. This way, we could omit the pylint comment as well. Perhaps @martynjames might give his opinion on this.